js/views/modals/Settings/Settings.js

```diff
@@ -75,104 +71,6 @@ export default class extends BaseModal {
     }
   }
 
-  saveClick() {
```

Comment:
This is code I forgot to remove when I was refactoring settings to integrate with the new modal design. Now tabs handle saving on their own.